docs/build.rb

```diff
@@ -0,0 +1,1143 @@
+require 'fileutils'
+require 'shellwords'
+require 'json'
```

Comment:
Can we confirm if this is still required?

Reply:
Compilation of the production site seems to still work despite this dependency and I'm able to load the pages and search fine without the `json`  dependency:

```
 ~/git/metasploit-framework/docs │ land-pr15923:pr/15923 !1 ?4  bundle exec ruby build.rb --production --serve
cd metasploit-framework.wiki; git pull

Have a nice day!

Already up to date.
JEKYLL_ENV=production jekyll build

Have a nice day!

Configuration file: /home/gwillcox/git/metasploit-framework/docs/_config.yml
            Source: /home/gwillcox/git/metasploit-framework/docs
       Destination: /home/gwillcox/git/metasploit-framework/docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
                    done in 39.053 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
[2021-12-13 18:10:28] INFO  WEBrick 1.7.0
[2021-12-13 18:10:28] INFO  ruby 3.0.2 (2021-07-07) [x86_64-linux]
[2021-12-13 18:10:28] INFO  WEBrick::HTTPServer#start: pid=103477 port=4000
127.0.0.1 - - [13/Dec/2021:18:11:48 CST] "GET / HTTP/1.1" 307 74
- -> /
127.0.0.1 - - [13/Dec/2021:18:11:48 CST] "GET /metasploit-framework/ HTTP/1.1" 200 54145
- -> /metasploit-framework/
127.0.0.1 - - [13/Dec/2021:18:11:48 CST] "GET /metasploit-framework/assets/js/just-the-docs.js HTTP/1.1" 200 15210
http://127.0.0.1:4000/metasploit-framework/ -> /metasploit-framework/assets/js/just-the-docs.js
127.0.0.1 - - [13/Dec/2021:18:11:48 CST] "GET /metasploit-framework/assets/js/toggle_mode.js HTTP/1.1" 200 1
http://127.0.0.1:4000/metasploit-framework/ -> /metasploit-framework/assets/js/toggle_mode.js
127.0.0.1 - - [13/Dec/2021:18:11:48 CST] "GET /metasploit-framework/assets/css/just-the-docs-default.css HTTP/1.1" 200 115241
http://127.0.0.1:4000/metasploit-framework/ -> /metasploit-framework/assets/css/just-the-docs-default.css
127.0.0.1 - - [13/Dec/2021:18:11:48 CST] "GET /metasploit-framework/assets/js/search-data.json HTTP/1.1" 200 941847
http://127.0.0.1:4000/metasploit-framework/ -> /metasploit-framework/assets/js/search-data.json
127.0.0.1 - - [13/Dec/2021:18:11:50 CST] "GET /metasploit-framework/docs/using-metasploit/getting-started/nightly-installers/ HTTP/1.1" 200 44218
http://127.0.0.1:4000/metasploit-framework/ -> /metasploit-framework/docs/using-metasploit/getting-started/nightly-installers/
127.0.0.1 - - [13/Dec/2021:18:11:52 CST] "GET /metasploit-framework/docs/using-metasploit/basics/how-to-use-a-metasploit-module-appropriately/ HTTP/1.1" 200 49473
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/getting-started/nightly-installers/ -> /metasploit-framework/docs/using-metasploit/basics/how-to-use-a-metasploit-module-appropriately/
127.0.0.1 - - [13/Dec/2021:18:12:01 CST] "GET /metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/ HTTP/1.1" 200 73710
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/basics/how-to-use-a-metasploit-module-appropriately/ -> /metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/
127.0.0.1 - - [13/Dec/2021:18:12:02 CST] "GET /metasploit-framework/assets/css/just-the-docs-default.css HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/ -> /metasploit-framework/assets/css/just-the-docs-default.css
127.0.0.1 - - [13/Dec/2021:18:12:02 CST] "GET /metasploit-framework/assets/js/just-the-docs.js HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/ -> /metasploit-framework/assets/js/just-the-docs.js
127.0.0.1 - - [13/Dec/2021:18:12:02 CST] "GET /metasploit-framework/assets/js/toggle_mode.js HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/ -> /metasploit-framework/assets/js/toggle_mode.js
127.0.0.1 - - [13/Dec/2021:18:12:02 CST] "GET /metasploit-framework/assets/js/search-data.json HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/ -> /metasploit-framework/assets/js/search-data.json
127.0.0.1 - - [13/Dec/2021:18:12:11 CST] "GET /metasploit-framework/docs/using-metasploit/advanced/meterpreter/meterpreter-timeout-control/ HTTP/1.1" 200 50634
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/python-extension/ -> /metasploit-framework/docs/using-metasploit/advanced/meterpreter/meterpreter-timeout-control/
127.0.0.1 - - [13/Dec/2021:18:12:11 CST] "GET /metasploit-framework/assets/css/just-the-docs-default.css HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/meterpreter-timeout-control/ -> /metasploit-framework/assets/css/just-the-docs-default.css
127.0.0.1 - - [13/Dec/2021:18:12:11 CST] "GET /metasploit-framework/assets/js/just-the-docs.js HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/meterpreter-timeout-control/ -> /metasploit-framework/assets/js/just-the-docs.js
127.0.0.1 - - [13/Dec/2021:18:12:11 CST] "GET /metasploit-framework/assets/js/toggle_mode.js HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/meterpreter-timeout-control/ -> /metasploit-framework/assets/js/toggle_mode.js
127.0.0.1 - - [13/Dec/2021:18:12:11 CST] "GET /metasploit-framework/assets/js/search-data.json HTTP/1.1" 304 0
http://127.0.0.1:4000/metasploit-framework/docs/using-metasploit/advanced/meterpreter/meterpreter-timeout-control/ -> /metasploit-framework/assets/js/search-data.json
```